### PR TITLE
Implement rn bootsplash

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="splashscreen_bg">#fff</color>
+    <color name="bootsplash_background">#FFFFFF</color>
 </resources>
```
Comment (marek-siemieniuk-morawski) [inline android/app/src/main/res/values/colors.xml:4]: Newline

File: src/state/authentication/sagas.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import SplashScreen from 'react-native-splash-screen';
+import RNBootSplash from "react-native-bootsplash";
```
Comment (marek-siemieniuk-morawski) [inline src/state/authentication/sagas.ts:1]: double `""` instead of a single quotation mark

**Comment by KedziaPawel** [comment]
@mateuszpmroz do you have linter set up properly?

**Comment by marek-siemieniuk-morawski** [comment]
As far as I understand someone with macOS should takeover it and finish implementation?

File: src/state/authentication/sagas.ts (hunk 2 above)
Comment (mateuszpmroz) [reply src/state/authentication/sagas.ts:1]: > As far as I understand someone with macOS should takeover it and finish implementation?

yes

**Comment by mateuszpmroz** [comment]
> As far as I understand someone with macOS should takeover it and finish implementation?

Yes, Kamil

**Comment by mateuszpmroz** [comment]
> @mateuszpmroz do you have linter set up properly?
@KedziaPawel 
In project we have `eslint fix` on commit and localy I have prettier and eslint on VSC, so I have errors and warnings. I'm not sure why `eslint fix` on precommit does not fix the quotation mark.

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**